themes/tiddlywiki/snowwhite/base.tid

```diff
@@ -36,7 +36,8 @@ tags: [[$:/tags/Stylesheet]]
 	<<filter "drop-shadow(0px -1px 2px rgba(0,0,0,0.25))">>
 }
 
-.tc-tiddler-frame input.tc-edit-texteditor {
+.tc-tiddler-frame input.tc-edit-texteditor,
+.tc-tiddler-frame select.tc-edit-texteditor {
```

Comment:
> I am not convinced that we should add the core CSS classes to support the field viewer being a "select" element. The user experience is confusing if the field contains a value that is not a tiddler title.

What do you mean by confusing? Could you please give me an example?

The style will only be applied to select and input elements that have the `tc-edit-texteditor` class. I don't think there is a problem with field editors that have multiple input elements, because these would not have the `tc-edit-texteditor` class assigned.

Without extending the style using a select-element looks weird:

![WithoutSelecCSS](https://user-images.githubusercontent.com/10105345/157110038-0fca4105-3a8e-4cb6-a431-bf31add8d4cc.png)

How should the user apply the styles to the select element to make it look correctly themed if we remove the select element from the `tc-edit-texteditor` class?